test/parallel/test-regress-GH-io-1151.js

```diff
@@ -0,0 +1,14 @@
+var assert = require('assert')
```

Comment:
I know the original test didn't have semicolons, but I think that we should add them to conform to style, no?